3.10/deployment-cluster-manual-start.md

```diff
@@ -310,25 +322,45 @@ arangod --server.authentication=false \
   --cluster.agency-endpoint tcp://192.168.1.1:8531 \
   --cluster.agency-endpoint tcp://192.168.1.2:8531 \
   --cluster.agency-endpoint tcp://192.168.1.3:8531 \
-  --database.directory dbserver &
-  
+  --database.directory dbserver
+
 arangod --server.authentication=false \
   --server.endpoint tcp://0.0.0.0:8529 \
   --cluster.my-address tcp://192.168.1.4:8529 \
   --cluster.my-role COORDINATOR \
   --cluster.agency-endpoint tcp://192.168.1.1:8531 \
   --cluster.agency-endpoint tcp://192.168.1.2:8531 \
   --cluster.agency-endpoint tcp://192.168.1.3:8531 \
-  --database.directory coordinator &
+  --database.directory coordinator
+```
+
+**Note 3:** you can easily extend the Cluster, by adding more machines which runs
+an _Agent_. For instance, if you have an additional forth
+machine with IP 192.168.1.5, you can execute the following commands
+
+On 192.168.1.5:
+
+```
+arangod --server.endpoint tcp://0.0.0.0:8531 \
+  --agency.my-address tcp://192.168.1.5:8531 \
+  --server.authentication false \
+  --agency.activate true \
+  --agency.size 4 \
+  --agency.endpoint tcp://192.168.1.1:8531 \
+  --agency.endpoint tcp://192.168.1.2:8531 \
+  --agency.endpoint tcp://192.168.1.3:8531 \
+  --agency.endpoint tcp://192.168.1.5:8531 \
+  --agency.supervision true \
+  --database.directory agent
 ```
```

Comment:
This is not true. We do not support changing the number of agents after deployment. This is going to cause serious trouble down the road, since the agency size must be constant (and equal) across all machines. Please remove this suggestion "Note 3".

Reply:
Ah yea. I learned that the hard way this week. So then what do you do if you need to increase the size of the agency or switch an exist agent to a new machine/ different IP address?